Extension/lib/webrequest.js

```diff
@@ -147,6 +147,13 @@
             return;
         }
 
+        // And we don't need this check on newer than 58 chromes anymore
+        // https://github.com/AdguardTeam/AdguardBrowserExtension/issues/572
+        if (adguard.utils.browser.isChromeBrowser() &&

```

Comment:
But with check is only performed for `document` and `subdocument` types.
And actually do we really need this case, cause this hack with modifying CSP headers had been implemented as the last approach only, then we have no other way to catch websockets requests.